charts/karpenter/values.yaml

```diff
@@ -5,6 +5,13 @@ serviceAccount:
   name: karpenter
   # -- Annotations to add to the service account (like the ARN of the IRSA role)
   annotations: {}
+options:
+  # -- Required. Cluster endpoint
+  clusterEndpoint: ""
+  # -- Required. Cluster name
+  clusterName: ""
+  # -- Required. The default instance profile to use when launching nodes on AWS
```

Comment:
If required, do we need to include the required directive in the chart?

Reply:
That's kinda the question right now... should it be required (see previous thread). There's obviously a mismatch here that would need to be fixed either way. Good catch on that

Reply:
Fixed.